### DISCUSSION TITLE
Aren't "split horizontally" and "split vertically" the wrong way around?

### DISCUSSION THREAD

**Discussion by chambers** [wezterm/wezterm#4291] [General]
Title.

Expected behaviour for split horizontally: Split "along the horizon" (x axis), such that two panes will be stacked on top of one another ("Top/Bottom")

Expected behaviour for split vertically: Split along the y axis, such that two panes will be side to side ("Left/Right")

It's like that in other programs e.g. Helix Editor. Not checked vim but I'm sure it would be the same.

Nobody seems to have noticed this so I could be wrong. I wanted to ask here first before I raise an issue or do a PR

**Comment by wez** [comment]
The terminology was inherited from iterm2 where it refers to the direction of the split rather than the positioning of the panes.
I think depending on the kind of person you are, either way around either makes perfect sense of feels wrong.

In the CLI and in the command palette, we use clarifying names like `--left` and `Left/Right` to refer to the positions of the resulting panes:

![image](https://github.com/wez/wezterm/assets/117777/cc98f1dc-3c92-4815-b713-9169401ceb07)

We can't simply swap the sense of the key assignment actions around without breaking everyone's existing config.

**Reply by chambers** [reply]
> it refers to the direction of the split

This is what I mean though, it's doesn't at all, it's the exact opposite. If I asked someone to cut a cake vertically I expect to get a piece of cake 🍰 and not an entire layer of the cake. 

Anyway it's a small thing, but it feels wrong because it is. Thanks for the solid terminal.
